### PR TITLE
feat: `default-keymap-open` and `current-keymap-open` typeable commands

### PR DESCRIPTION
**Pull request by AlexanderBrevig**
`:current-keymap-open` will open a new buffer with the currently loaded config. 
`:default-keymap-open` will open a new buffer with the default config.

### CODE REVIEW

**Comment by AceofSpades5757** [comment]
When are you getting this `ValueAfterTable` error? I'm not getting this error.

**Comment by AlexanderBrevig** [comment]
> When are you getting this `ValueAfterTable` error? I'm not getting this error.

We very quickly sorted it out. I'll update the description.

**Comment by AceofSpades5757** [comment]
It's cool, to be sure, but I'm not sure what the purpose would be used for. Would I want to fill a buffer full of all the key mappings? If I change a mapping, and save it, will it change (it won't but this was something I tried)?

I think it would be cool if I could specify a key map, and it would place that configuration in my primary register. That might be a bit out of scope though for what you're trying to accomplish.

**Comment by AceofSpades5757** [comment]
Adding the different modes was a nice touch. Seeing all the different keys while in the different mode is wild to see.

**Comment by s0LA1337** [comment]
> It's cool, to be sure, but I'm not sure what the purpose would be used for. Would I want to fill a buffer full of all the key mappings?

Let me explain how at least I had the idea of this feature.
I am working at home with QWERTY and at work with Colemak, changing every setting here to Colemak is kinda a pain. 
The idea was, that this will make a few things easier. 
- when you want to remap keys
- debugging as in e.g. To see whether you have keys which overlap the default configuration 
- generally, I also think it's a good idea to have some way to access the default configuration

**Comment by AlexanderBrevig** [comment]
Maybe it should be `open-current-keymap` that shows the active working set, and then a `open-default-keymap` as well? 
We should set the buffer to read-only.

**Comment by AlexanderBrevig** [comment]
There may be a bug with it printing duplicate config. Will investigate.

**Comment by AlexanderBrevig** [comment]
Since select mode inherits normal mode, we should remove duplicates before serializing to keep output more similar with `keymap/default.rs`

File: helix-term/src/commands/typed.rs [hunk 1]
```diff
@@ -1424,6 +1427,87 @@ fn open_log(
...
+        // there must be a way to make this easier i think
+        .map(|couple| match couple.0 {
+            Mode::Insert => toml::to_string(&Keys {
+                keys: ModeInsert { insert: couple.1 },
+            }),
+            Mode::Normal => toml::to_string(&Keys {
+                keys: ModeNormal { normal: couple.1 },
+            }),
+            Mode::Select => toml::to_string(&Keys {
+                keys: ModeSelect { select: couple.1 },
```
Comment (AlexanderBrevig) [inline helix-term/src/commands/typed.rs:1460]: We should remove duplicates from normal mode here

**Comment by s0LA1337** [comment]
related issue: #1210

**Pull request closed without merging**